lib/v2/onehu/radar.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+    'onehu.xyz': {
+        _name: 'Onehu',
```

Comment:
https://github.com/DIYgod/RSSHub/blob/eec19e594c95b89776b171831fda39ffa57f5d31/website/docs/routes/new-media.mdx#L5292
```suggestion
        _name: '我不是盐神',
```